### ISSUE TITLE
Final Report Review hy488

### ISSUE THREAD

**Issue by haoqiyao** [nancy15689/SentimentAnalysisTweets#12]
The project is about predicting people's view towards certain subject from mining twitter data. Their data contains positive and negative tweets, and each tweet has date, author and the strings in it. The best result they produced is about 70% and in general it is a NLP problem.

The things I like about this project is that they provided many explanations for their approaches. Data cleaning, models usage are supported by reasons and readers can understand their thoughts from them. Moreover, the team provided innovative model which is different from what the class taught, "Ngram model". It is cool to learn a new model from reading their paper. At last, I enjoyed the part how they used regularizers to help the model generalized.

One of the things can be improved in this project is that they can use more formulas or graphs to explain models, like to the Ngram model, instead of many words, because it is hard to read all the words especially they already had a dense introduction and background part. The next thing is that the format of the report is really tight, I suggest to have more space towards each line. Finally, as some of the classmates might not understand NLP models, it is better to have a part to explain basic NLP approach instead of just giving us the model right away.

In conclusion, the team did a great job and even though the result for each model were the same, their approach were well delivered from this paper.
